browser/src/map/handler/Map.StateChanges.js

```diff
@@ -54,6 +54,14 @@ L.Map.StateChangeHandler = L.Handler.extend({
 				$('.leaflet-pane.leaflet-map-pane').removeClass('bucket-cursor');
 		}
 
+		if (e.commandName === '.uno:StartWithPresentation' && (state === true || state === 'true')) {
+			let startPresentationParam = window.coolParams.get('startPresentation');
+			if (startPresentationParam === undefined || (startPresentationParam === 'true' || startPresentationParam == true)) {
```

Comment:
```suggestion
			if (typeof startPresentationParam === 'undefined' || (startPresentationParam === 'true' || startPresentationParam == true)) {
```

Reply:
In fact window.coolParams.get return '' when the key is missing, defined in global.js